overlord/fdestate/backend/reseal.go

```diff
@@ -62,6 +62,49 @@ type StateUpdater func(role string, containerRole string, bootModes []string, mo
 
 // ResealKeyForBootChains reseals disk encryption keys with the given bootchains.
 func ResealKeyForBootChains(updateState StateUpdater, method device.SealingMethod, rootdir string, params *boot.ResealKeyForBootChainsParams, expectReseal bool) error {
+	return resealKeyForBootChains(updateState, method, rootdir,
```

Comment:
Should we change more than one letter in the name?

Reply:
renamed that internal function to simply `resealKeys`